app/app.js

```diff
@@ -92,7 +95,7 @@ const render = (messages) => {
 if (module.hot) {
   // modules.hot.accept does not accept dynamic dependencies,
   // have to be constants at compile-time
-  module.hot.accept('./i18n', () => {
+  module.hot.accept(['./i18n', 'containers/App'], () => {
```

Comment:
I don't this this is sufficient without also calling `require` on `App` in the callback. At least, it didn't work for me until I added that (using 3.4 and RHL@next branch).